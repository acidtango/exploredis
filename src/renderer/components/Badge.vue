<template>
  <span>
    <slot />
  </span>
</template>

<script>
export default {
  name: 'Badge',
}
</script>

<!-- Add "scoped" attribute to limit CSS to this component only -->
<style scoped lang="scss" scoped>
span {
  align-items: center;
  background: #8983be;
  border-radius: 2px;
  color: white;
  display: flex;
  font-family: -apple-system, BlinkMacSystemFont, Roboto, sans-serif;
  font-size: 10px;
  font-weight: 600;
  height: 18px;
  justify-content: center;
  min-width: 36px;
}
</style>

<docs>
This badge is amazing, use it responsibly.

## Examples

```jsx
<Badge>String</Badge>
```
</docs>
