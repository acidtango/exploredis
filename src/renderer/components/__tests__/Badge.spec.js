import { mount } from '@vue/test-utils'
import Badge from '@/components/Badge.vue'

describe('<Badge />', () => {
  it('renders children', () => {
    const slots = {
      default: ['STR'],
    }
    const wrapper = mount(Badge, { slots })
    expect(wrapper).toMatchSnapshot()
  })
})
