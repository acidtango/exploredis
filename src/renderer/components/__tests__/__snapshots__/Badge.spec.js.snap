// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<Badge /> renders children 1`] = `<span class="badge">STR</span>`;
